applications/serial_lte_modem/doc/HTTPC_AT_commands.rst

```diff
@@ -207,48 +208,53 @@ Syntax
   It represents the length of a partially received HTTP response.
 * The ``<state>`` value can assume one of the following values:
 
-  * ``0`` - The entire HTTP response has been received.
-  * ``1`` - There is more HTTP response data to come.
+  * ``0`` - There is more HTTP response data to come.
+  * ``1`` - The entire HTTP response has been received.
 
 * The ``<response>`` is the raw data of the HTTP response, including headers and body.
 
 Example
 ~~~~~~~
 
-The following example sends a POST request to send data to the server with an optional header.
+The following example sends a PUT request to send JSON format data to the server with an optional header.
 
 ::
 
-   AT#XHTTPCREQ="POST","/post","User-Agent: SLM/1.2.0
-   Accept: */*
-   Content-Type: application/x-www-form-urlencoded
-   Content-Length: 20
-   ",20
+   at#xhttpccon=1,"iot.cht.com.tw",80
```

Comment:
```suggestion
   AT#XHTTPCCON=1,"iot.cht.com.tw",80
```

Reply:
Can we use example.com as URL?

Reply:
Yes, example.com  would be more suitable. I've modified it, thanks.